responsible-ai-collaborative/aiid
# Related Incident displaying issue reports (breaks submission)

- closes #2594 

Issue reports (orphan reports) were being displayed as related incidents which broke submission when choosing a relation option.

On related incidents, only reports from incidents should appear now. Submission should work as expected.

To test you can use this flow:

URL: https://arstechnica.com/tech-policy/2024/01/fake-ai-taylor-swift-images-flood-x-amid-calls-to-criminalize-deepfake-porn/
<<Fetch, keeping all entries as automatically filled except the following>>
Marked as unrelated (no):
"#348 - YouTube algorithm pushed election fraud claims to Trump supporters, report says"
Marked as related (yes):
**"- Thousands scammed by AI voices mimicking loved ones in emergencies" --> Should not appear**
"#597 - Teen boys use AI to make fake nudes of classmates, sparking police probe"
Publication date (marked as related):
"#626 - Taylor Swift fans swindled by AI-generated Le Creuset ad scam"
Incident Date
01/26/2024
Submitters (automatically associated with my user account): "Sean McGregor"
Incident Title
"Lorem Ipsum Doler"
Description
"Lorem Ipsum Doler"
Alleged Deployer (suggested): "Twitter Users"
Alleged Developer (suggested): "Unknown"
Alleged harmed or nearly harmed parties: "Taylor Swift"
<Submitted>

Test URL: https://pr-2595--staging-aiid.netlify.app/apps/submit/
